public/sites/default/env.indicator.settings.php

```diff
@@ -0,0 +1,27 @@
+<?php 
+/**
+ * The UI for environment indicator is broken, so we have to do this instead.
+ * See more info, https://www.drupal.org/project/environment_indicator/issues/2224983
+ */
+switch ($_SERVER['HTTP_HOST']) {
+	case 'edit.infofinland.fi/':
+		$config['environment_indicator.indicator']['bg_color'] = '#ffffff';
+		$config['environment_indicator.indicator']['fg_color'] = '#000000';
+		$config['environment_indicator.indicator']['name'] = 'Production';
+		$break;
```

Comment:
$break?

Actually interesting if this would work, I would expect that this just defines a variable called $break and without the break; it would run straight along to the next case and overwrite the $config variables and all you'd end with in the end is the last set.